x-pack/plugins/canvas/server/routes/kibana.js

```diff
@@ -0,0 +1,45 @@
+/*
```

Comment:
feels we are duplicating code to get a list of saved visualizations

Reply:
We want as little reliance on Kibana imports as possible. I was hoping to remove the render one too, but that thing is a rabit hole so I had to leave it alone.

Reply:
ok this doesn't convince me, it means you rewrite a lot of code where you could depend on kibana, This is a nightmare when it comes to maintinance, as you need to fix stuff in multiple places etc .... in my opinion we should reuse as much as possible, but thats just my opinion.

the thing is that when we'll change anything in visualize canvas will be broken as its doing stuff its own way. We'' also get a better chance to hit scenarios where we'll be adding features to kibana and they just won't propagate to canvas (the same thing we have today with timelion or tsvb)

Reply:
If we change how we query saved objects in Kibana, a lot of other stuff is going to break. But, fair enough. I mostly wrote the endpoint before I realized I could get away from the ui import. 

That said, it would be amazing if this moved to a backend route in Kibana. I basically just took what's in the client code and put it in a server route, so it's not like it's a hard thing to do.